_projects/2023/100407614/100407614.Rmd

```diff
@@ -0,0 +1,581 @@
+  ) +
+  annotate(
+    "text",
+    x = Inf, y = 10,
+    label = "50",
+    vjust = -0.5, hjust = 1, size = 6, color = "#CC0000", family = "font_y_axis"
+  ) +
+  annotate(
+    "text",
+    x = Inf, y = 11,
+    label = "55",
+    vjust = -0.5, hjust = 1, size = 6, color = "#CC0000", family = "font_y_axis"
+  )
+cross_chart + theme(plot.title = element_markdown())
+
+```
+
+## Incorporating annotations on the most relevant dates
+
+Annotations are included using annotate("text"), and arrows are added with annotate("curve" or "segment"), depending on the shape of the arrow. The arrow command is used to specify the dimension and type of the arrowhead, in this case, it is a closed arrow. Finally, lineheight is used to determine the spacing between lines.
+
+```{r fig.height=6, fig.width=9}
+cross_chart + theme(plot.title = element_markdown()) +
+  annotate("curve", x= 1926.5, xend = 1929, y=5, yend= 4.4,
+           curvature = -0.2, arrow = arrow(type = "closed", length = unit(0.015, "npc"), angle = 20)) +
+  annotate("text", x=1926, y=5, hjust=1, label ="1929: The Great Depression", color ="#5b5b5b", size = 5, family = "font_y_axis") +
+  annotate("curve", x= 1941.5, xend = 1944.5, y=9.2, yend= 8.2,
+           curvature = -0.2, arrow = arrow(type = "closed", length = unit(0.015, "npc"), angle = 20)) +
+  annotate("text", x=1941, y=9.2, hjust=1, label ="1945: The Second World War ends", color ="#5b5b5b", size = 5, family = "font_y_axis") +
+  annotate("curve", x= 1946.5, xend = 1949, y=10.6, yend= 9.6,
+           curvature = -0.2, arrow = arrow(type = "closed", length = unit(0.015, "npc"), angle = 20)) +
+  annotate("text", x=1946, y=10.5, hjust=1, label ="In the late 1940s and early 50s epidemiologists\n establish that smoking is harmful to people's health", color ="#5b5b5b", size = 5, lineheight = 0.75, family = "font_y_axis") +
+  annotate("curve", x= 1960.5, xend = 1964, y=11.65, yend= 10.9,
+           curvature = -0.5, arrow = arrow(type = "closed", length = unit(0.015, "npc"), angle = 20)) +
+  annotate("text", x=1960, y=11.5, hjust=1, label ="1964: Surgeon General's report links smoking\n to deaths from cancer and heart disease", color ="#5b5b5b", size = 5, lineheight = 0.75, family = "font_y_axis") +
+  annotate("curve", x= 1968.5, xend = 1970, y=12.2, yend= 10.3,
+           curvature = -0.2, arrow = arrow(type = "closed", length = unit(0.015, "npc"), angle = 20)) +
+  annotate("text", x=1968, y=12.2, hjust=1, label ="1970: the US bans cigarette ads on radio and television", color ="#5b5b5b", size = 5, family = "font_y_axis") +
+  annotate("segment", x= 1983, xend = 1983, y=8.35, yend= 9,
+           arrow = arrow(type = "closed", length = unit(0.015, "npc"), angle = 20)) +
+  annotate("text", x=1983, y=8.3, hjust=0.9, vjust=1, label ="1983: federal tax on\n cigarettes doubles", color ="#5b5b5b", size = 5, lineheight = 0.75, family = "font_y_axis") +
+  annotate("segment", x= 1986, xend = 1986, y=7.05, yend= 8.4,
```

Comment:
Please take a look at the output document. Some of the lines are too wide. Restyle all your code so that it fits in the column. A maximum of 80 characters per line is generally advisable, so that the code is readable.